test/Microsoft.AspNetCore.Authentication.Test/FacebookTests.cs

```diff
@@ -585,7 +585,7 @@ private void ConfigureDefaults(FacebookOptions o)
             var transaction = await server.SendAsync("http://example.com/base/login");
             Assert.Equal(HttpStatusCode.Redirect, transaction.Response.StatusCode);
             var location = transaction.Response.Headers.Location.AbsoluteUri;
-            Assert.Contains("https://www.facebook.com/v2.6/dialog/oauth", location);
+            Assert.Contains("https://www.facebook.com/v2.12/dialog/oauth", location);
```

Comment:
Nit: Assert using the constants?

Reply:
I could go either way on that. There are pros/cons to using the fields vs. repeating the data.

Reply:
Yeah but in this case isn't it basically a always bug if these ever are inconsistent since we are exposing this in our Defaults class?

Reply:
Pros to using const's:
* Less work to maintain because you make the change in one place

Cons to using const's:
* Less clear when you make a src change to see what effect it has on the product


So, yes, it'd be a bug if it's inconsistent, and this test would tell you that, and you have to fix it (of course). Using a const would avoid the need to have to change the test.

Reply:
But mainly the upside is it makes future updates trivial, as its just updating the constant

Reply:
And that's why I'm fine either way 😄

Reply:
I actually prefer updating it separately in the tests. It's like having to enter your e-mail address twice to confirm you did it right.